### ISSUE TITLE
Can't seem to build with instructions in Contributing.md

### ISSUE THREAD

**Issue by ericswpark** [jellyfin/Swiftfin#465]
Hi, I'm trying to build Swiftfin from source by following the instructions in Contributing.md. However, when the project workspace opens in Xcode I get the following error:

<img width="481" alt="image" src="https://user-images.githubusercontent.com/13326074/174443810-ad620f2c-f226-499b-bf04-25e52b00f962.png">

> Source files for target ActivityIndicator should be located under 'Sources/ActivityIndicator', or a custom sources path can be set with the 'path' property in Package.swift

When I attempt to build I get 22 additional errors, maybe because the ActivityIndicator error hasn't been resolved yet and Xcode can't build the dependency graph?

<img width="480" alt="image" src="https://user-images.githubusercontent.com/13326074/174443885-50ac450e-89ba-4ec3-b4d1-e419296bd213.png">

I've followed the instructions to install `carthage` and fetch the dependencies, and have also tried cleaning the build folder and retrying the build. However the initial error just wouldn't resolve, so I don't know what I'm missing. Any help would be appreciated!

I'm using Xcode 13.4.1 (13F100) on macOS Monterey 12.4 (21F79).

**Comment by LePips** [comment]
These are quirks with Swift Package Manager. The only thing I can recommend I closing and re-opening Xcode as it redownloads the packages if necessary (sometimes it misses one, in this case ASCollectionView).

**Comment by PangMo5** [comment]
If the problem persists after restarting the Xcode, 
try the following steps:
1. reset spm caches (File -> Packages -> Reset Package Caches)
2. clean build folder (cmd + shift + k)
3. try build

**Comment by ericswpark** [comment]
Thanks everyone! Restarting Xcode fixed the issue.

**Comment by kid1412621** [comment]
How can I resole this? I reset and clean cache dozen of times.
```
Failed to resolve dependencies Dependencies could not be resolved because no versions of 'get' match the requirement 1.0.4..<2.0.0 and root depends on 'jellyfin-sdk-swift' 0.3.0..<1.0.0.
'jellyfin-sdk-swift' >= 0.3.0 practically depends on 'get' 1.0.4..<2.0.0 because 'jellyfin-sdk-swift' 0.3.0 depends on 'get' 1.0.4..<2.0.0 and no versions of 'jellyfin-sdk-swift' match the requirement 0.3.1..<1.0.0.
```

```
Failed to resolve dependencies Dependencies could not be resolved because 'pulseloghandler' depends on 'pulse' 4.0.0..<5.0.0 and root depends on 'pulse' 3.0.0..<4.0.0.
```

**Comment by LePips** [comment]
Thank you for bringing this up, as it turned out a lot bigger than I thought it would be.

Using my fork instead of the actual SDK was because that package was also being extensively worked on during a lot of work and just missed switching it over after everything was complete.

The latter error came from PulseLogHandler not having the correct tvOS minimum supported version to match with Pulse which I fixed and it has been updated.

Everything should be good now for fresh setups.
